allennlp/training/callbacks/callback_handler.py

```diff
@@ -0,0 +1,49 @@
+"""
+blah
+"""
+from typing import Iterable, Dict, NamedTuple, Callable, List
+from collections import defaultdict
+import inspect
+
+from allennlp.training.callbacks.callback import Callback
+
+
+class CallbackMember(NamedTuple):
+    callback: Callback
+    name: str
+    member: Callable
```

Comment:
Would it be better to call this something more specific like `event_handler`  instead of just `member`? Or maybe the named tuple could be called `CallbackEventHandler`? That way the naming would correspond to the `@event_handler` decorator

Reply:
I agree with this.

Reply:
Can't you be more specific here, though?  `event_handler: Callable[[TrainerBase], None]`